src/store/index.ts

```diff
@@ -16,12 +16,11 @@
  */
 import Vue from "vue";
 import Vuex from "vuex";
```

Comment:
do we need state management?

Reply:
Yes, we would require state management for this project.